src/ape/api/providers.py

```diff
@@ -884,6 +884,9 @@ def _eth_call(self, arguments: List) -> bytes:
         except Exception as err:
             raise self.get_virtual_machine_error(err) from err
 
+        if "error" in result:
```

Comment:
I think this already happens in `_make_request()`

Reply:
It has some sort of bug, because I was reaching the point where it tried to convert an error message dict `result` from RPC to `HexBytes` below

Reply:
hmm, weird, if you set a breakpoint, does it hit this part? https://github.com/ApeWorX/ape/blob/main/src/ape/api/providers.py#L1114-L1116

Reply:
Should probably figure out if this is a good thing to merge?